### PR TITLE
Guys, please review my initial version of the e-commerce

### PR DESCRIPTION
**Pull request by jugalkpatel**
- Manufacturer, Product Category filters are not working.
- at refresh front end is not completely consistent with the app. the new lunch app is always plain as there is no user.  but after add to cart or wishlist, it shows previously added items.

### CODE REVIEW

**Comment by asmitzz** [comment]
image
Fixed the sidebar manufacture options last 1 option is not visible

**Comment by asmitzz** [comment]
when the item is added in the cart change the color or highlight the wishlist icon so that user can know that item is already wishlisted

**Comment by asmitzz** [comment]
just a suggestion add loader for cart,wishlist,add qty,decrease qty buttons so that user can know something is happening there

**Comment by asmitzz** [comment]
when I choose any of the option from products category and manufacture filter then the white screen comes fixed this bug

**Comment by asmitzz** [comment]
fix netlify broken link issue if we refresh the page

**Comment by asmitzz** [comment]
just a suggestion card size is too big try to make it small and put less content

**Comment by asmitzz** [comment]
Just a suggestion make filter similar like when it shows on the small devices or override the filter page will be a good ux

**Comment by asmitzz** [comment]
overall ui is good and backend code is so clean and readable just remove commented console logs

File: src/components/FilterBar/FilterBar.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import "./FilterBar.css";
```
Comment (Ayush-kumar1) [inline src/components/FilterBar/FilterBar.js:2]: The filter function is broking the code.Make sure that your filter functions start working.

File: src/components/Header/Header.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import React from "react";
```
Comment (Ayush-kumar1) [inline src/components/Header/Header.js:1]: Your header looks very empty.Maybe add some heading(just a suggestion)

File: src/components/ProductCard/ProductCard.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
+import React from "react";
+
+import "./ProductCard.css";
+
+// import closeBtn from "../../assets/svgs/close-btn.svg";
+import arrow from "../../assets/svgs/right-arrow.svg";
```
Comment (Ayush-kumar1) [inline src/components/ProductCard/ProductCard.js:5]: Make your cards a bit more noticeable. Right now your card is almost similar to the background of the product page. Make it a bit more noticeable

File: src/components/ProductCard/WishListCard.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
+import React from "react";
```
Comment (Ayush-kumar1) [inline src/components/ProductCard/WishListCard.js:1]: Add add to cart button in the wishlist also. It's more convenient for the user

**Review by Ayush-kumar1 — COMMENTED** [review]
The code looks good. I have left few comments,pls go through it.

**Pull request closed without merging**